### PR TITLE
remove `parameters` from `Imports`; closes #10

### LINKED ISSUE #10: getting rid of `parameters` dependency
Functions where it is needed and can't be avoided.

- `principal_components`

https://github.com/easystats/datawizard/blob/6846aa40119d401039f79a8a7e85d7391857e26c/R/principal_components.R#L396

- `factor_analysis`

https://github.com/easystats/datawizard/blob/4c88f8d076ddd1823461e0c201f850672990e9ad/R/factor_analysis.R#L70-L88

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

**Comment by IndrajeetPatil** [comment]
> _Hello? (Hello? Hello? Hello?)
> Is there anybody in there?
> Just nod if you can hear me
> Is there anyone home?_

**Review by strengejacke — APPROVED** [review]

**Comment by IndrajeetPatil** [comment]
@DominiqueMakowski Still waiting for your approval, mate.

**Review by DominiqueMakowski — APPROVED** [review]

**Pull request merged**
